subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -462,7 +462,7 @@ We're now done with fixing the problems with this simple task.
 
 Keep reading to learn how to adopt the configuration cache for your build or your plugins.
 
-[[config_cache:task_out_out]]
+[[config_cache:task_opt_out]]
```

Comment:
The release notes still reference `task_out_out`.

Reply:
Hah! Thanks